README.md

```diff
@@ -113,6 +113,13 @@ To change the endpoints you can modifiy the AppSettings.cs file in the SmartHote
 ## Demo Scripts
 There is a detailed step-by-step demo script in the [demoscripts](https://github.com/Microsoft/SmartHotel360-mobile-desktop-apps/tree/master/DemoScripts) folder of this repo that you can use in your own presentations.
 
+## IoT Demo - Comming Soon
+
+1. In order to have live data working and enable room controls you must setup the [IoT Demo Backend](https://github.com/Microsoft/SmartHotel360-AKS-DevSpaces-Demo/blob/helm/README.md)
+2. Get Room Devices Api endpoint (setup via step 1) and change the value of DefaultRoomDevicesEndpoint in AppSettings.cs
```

Comment:
can we have **(setup via step #1)** be the same text in #s 2 & 3, one currently has a # and the other doesn't